scopes/workspace/workspace/workspace.ts

```diff
@@ -1253,6 +1256,60 @@ export class Workspace implements ComponentFactory {
       await this.importObjects();
       this.logger.consoleSuccess();
     }
+    return this.installModules(options);
+  }
+
+  private async getComponentsWithDependencyPolicies() {
+    const allComponentIds = await this.listIds();
+    const componentConfigFiles: ComponentConfigFile[] = [];
+    const componentPoliciesById: Record<string, any> = {};
+    (
+      await Promise.all<ComponentConfigFile | undefined>(
+        allComponentIds.map((componentId) => this.componentConfigFile(componentId))
+      )
+    ).forEach((component) => {
+      const depResolverAspect = component?.aspects.get(DependencyResolverAspect.id);
```

Comment:
ok but then how do I update the data in the component.json?

Reply:
ok I was confused a bit since the name in the forEach is component, but it's actually not a component but a componentConfigFile. 
In general, it will be better to have an API in the dep resolver which gets a componentConfigFile and knows to extract the config from it.

Reply:
I would rename it to depResoverConfig. it's not the aspect itself, it's only its config

Reply:
Done except adding an API to dep resolver which gets componentConfigFile